src/kernel/kernel.rs

```diff
@@ -67,11 +66,7 @@ where
                         kernel: akr.clone(),
                     };
 
-                    let mb = mailbox.clone();
-                    let d = dock.clone();
-
-                    let _ = std::panic::catch_unwind(AssertUnwindSafe(|| run_mailbox(mb, ctx, d)));
-                    //.unwrap();
+                    let _ = std::panic::catch_unwind(AssertUnwindSafe(|| run_mailbox(&mailbox, ctx, &mut dock))); //.unwrap();
```

Comment:
what's with the `.unwrap()` comment?

Reply:
That was already there before. I removed it because it isn't needed and it's just confusing.